### ISSUE TITLE
Feature Request : Package.json Babel RC support (instead of .babelrc)

### ISSUE THREAD

**Issue by SilentCicero** [parcel-bundler/parcel#762]
Parcel is awesome.

I would love if I could just use the package.json to specify my .babelrc settings. Currently, that doesn't seem to be supported, and I'm forced to include an additonal .babelrc file. I hate those little files. Any chance this could be an option like in webpack?

**Comment by SilentCicero** [comment]
Thoughts on this?

**Comment by marcklei** [comment]
On 1.5.1 configuring babel through package.json works for me. Do you have an example config which fails?

**Comment by marcklei** [comment]
Ok, on 1.5.1 it works, but on  current master this was removed with #559.

**Comment by sscaff1** [comment]
Why was this removed? How can I use babel-stage-2 or stage-0 or any other babel preset with 1.6?

**Comment by SilentCicero** [comment]
A babel rc file ATM.

Yeah, I would really like this added back, package Json is an acceptable place for babel settings almost everywhere else in the ecosystem. Parcel should accommodate.

**Comment by sscaff1** [comment]
@SilentCicero how did you get this to work with a babelrc file? I tried the babelrc file approach and it still blew up when trying to use features that required stage-2. I went back to 1.5 and everything works.

**Comment by SilentCicero** [comment]
Isn't stage 0/1 further ahead than stage 2?

Maybe a deer ate issue.

**Comment by sscaff1** [comment]
@SilentCicero yes. My project is just using stage-2 though and I'm unable to use any stage-2 features. From what I understand of the 1.6 release notes, babel-preset-env is automatic but how do I add to that?

**Comment by devongovett** [comment]
Babel config in package.json *should* already work. See:

https://github.com/parcel-bundler/parcel/blob/595252c4135c9f9d6cbb8aa5f3f2e94771cc9712/src/transforms/babel.js#L193-L199

**Comment by hrldcpr** [comment]
See #927 for an example of babel config in package.json not working
